app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -16,6 +16,11 @@
 
 package com.google.droidjump;
 
+import static com.google.droidjump.GameConstants.GAME_VIEW_CURRENT_LEVEL_STRING;
+import static com.google.droidjump.GameConstants.GAME_VIEW_DATA;
+import static com.google.droidjump.GameConstants.GAME_VIEW_LAST_LEVEL_STRING;
```

Comment:
nit: separe the static imports with a new line from the other types of imports.

Reply:
Fixed in all files.